plugins/MeteorShowers/src/MeteorShower.cpp

```diff
@@ -491,6 +491,7 @@ QString MeteorShower::getSolarLongitude(QDate date)
 	// put it in the range 0 to 360 degrees
 	l /= 360.;
 	l = (l - static_cast<int>(l)) * 360. - 1.;
```

Comment:
what does this line do? This and the previous seem to do a modulo 360, but why subtract 1?

Reply:
Feel like they tried to workaround to make it aligned with present day. Anyway, as I said above, this is not the right way to calculate annual meteor showers.